README.md

```diff
@@ -42,8 +49,28 @@ Install all dependencies and the project:
 poetry install
 ```
 
-Start the server in simulation mode:
+## Start the server
+
+### Either start the server directly on your host
 
 ```
-python3 -m nfv_test_api.main --config config.yaml --simulate
-```
\ No newline at end of file
+python3 -m nfv_test_api.main --config config.yaml
+```
+
+### Or start the server within container
+
+We first need to build the docker image (make sure your current directory is the root of the repo, where the dockerfile is located):
+
+```
+sudo docker build -t nfv-test-api .
+```
+
+To run a container with the image:
+
+```
+sudo docker run -d nfv-test-api
+```
```

Comment:
Same here.

Also:
1. it would be better to give a name to the container, so that you can show how to get the ip of it
2. you need to mention that, in order to use all the functionalities of the container, it needs to run with the privileged flag set to true

```
guillaume@latitude-5411:~$ docker run -d --name test --privileged inmantaci/nfv-test-api:latest
200ee25210ff9f978242cd641c8bee22fe0ee33ea63c9eea448cc1a4040fba30
guillaume@latitude-5411:~$ docker inspect test -f "{{ .NetworkSettings.Networks.bridge.IPAddress }}"
172.17.0.2
```